stubs/ArrayObject.stub

```diff
@@ -85,3 +85,22 @@ class ArrayObject implements IteratorAggregate, ArrayAccess
 	public function setIteratorClass($iterator_class) { }
 
 }
+
+/**
+ * @template TValue
+ * @implements Iterator<int, TValue>
+ * @implements ArrayAccess<int, TValue>
+ */
+class SplFixedArray implements Iterator, ArrayAccess, Countable
+{
+	/**
+	 * @param array<int, TValue> $array
```

Comment:
It would be more correct to have `@template T` (something else then TValue) above this method and use that in its signature.

Reply:
oh of course.